build.sbt

```diff
@@ -3,22 +3,27 @@ import com.gu.riffraff.artifact.RiffRaffArtifact.autoImport._
 import com.typesafe.sbt.packager.archetypes.systemloader.ServerLoader.Systemd
 import play.sbt.PlayImport.PlayKeys._
 import sbt.Keys.libraryDependencies
+import sbt.util
 
 import scala.concurrent.duration.DurationInt
 
 // common settings (apply to all projects)
-organization in ThisBuild := "com.gu"
-version in ThisBuild := "0.2.0"
-scalaVersion in ThisBuild := "2.12.15"
-scalacOptions in ThisBuild ++= Seq("-deprecation", "-feature", "-unchecked", "-target:jvm-1.8", "-Xfatal-warnings")
```

Comment:
Did I accidentally remove the target jvm argument?

Reply:
My mistake! I've re-added now, thanks!

Reply:
No, I did this on purpose and I found it's required for the project to compile.

Reply:
Ah, it works for me with the flag still there. I wonder if it's something to do with your local java version. Happy to pair on it?